src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -1092,4 +1092,10 @@ Non-HTTPS access will be removed in a future version. Consider migrating to 'HTT
     <value>NuGetAudit is enabled, but no package sources contain known vulnerability data.</value>
     <comment>Do not translate NuGetAudit</comment>
   </data>
+  <data name="Error_InvalidNuGetAuditValue" xml:space="preserve">
+    <value>Invalid NuGetAudit value '{0}'. Expected values: {1}</value>
```

Comment:
Should it be, allowed instead or valid values instead of expected?

Expected <-> Actual, Valid <-> Invalid is my thinking.